src/nvim/testdir/Makefile

```diff
@@ -101,7 +101,10 @@ test1.out: $(NVIM_PRG)
 $(SCRIPTS): $(NVIM_PRG) test1.out
 
 NO_PLUGINS = --noplugin --headless
-NO_INITS = -U NONE $(NO_PLUGINS)
+# In vim, if the -u command line option is specified, compatible is turned on
+# and viminfo is not read. Unlike vim, neovim reads viminfo and requires the
+# -i command line option.
+NO_INITS = -U NONE -i viminfo $(NO_PLUGINS)
```

Comment:
Makes sense.
But why not `-i NONE`?

Reply:
About "-i NONE": @justinmk you've used "-i viminfo" in 61fccda19 explicitly.  Any input?

Comment from there:

> Ostensibly this change should not be necessary, because of this
statement in runtest.vim: set viminfo+=nviminfo

Reply:
> > Ostensibly this change should not be necessary, because of this
> > statement in runtest.vim: set viminfo+=nviminfo

`viminfo+=nviminfo` is equivalent to `-i viminfo`, so I was only following that existing pattern.

Reply:
ok.
I think it makes sense to use `-i NONE` for NO_INIT.

(Still unclear though why you had to add it in the first place)

Reply:
> (Still unclear though why you had to add it in the first place)

Because `test_marks.res` is  affected by shada/viminfo. Or do you mean why didn't `runtest.vim` handle that already? (In that case: I don't know :)

Reply:
Yes, the latter.. :)